tutorials/preprocessing/20_plotting_projectors.py

```diff
@@ -0,0 +1,82 @@
+# -*- coding: utf-8 -*-
+"""
+.. _projectors-topomap-tutorial:
+
+Plotting the spatial pattern of projectors
+==========================================
+
+.. include:: ../../tutorial_links.inc
+
+This tutorial describes how to plot the topography (spatial pattern) of a
+projector. As usual we'll start by importing the modules we need, and loading
```

Comment:
Note that most EEG folks probably don't know what projectors are :) Maybe add a few nice friendly sentences about referencing schemes here, or just tell EEG folks to skip this.